<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shortcut</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="2" colspan="2">
    <widget class="QLineEdit" name="commentLineEdit">
     <property name="text">
      <string>Shortcut Created by Yawpm (Yet Another Wine Prefix Manager)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QLineEdit" name="categoryLineEdit">
     <property name="toolTip">
      <string>The location in the applications menu that you want the shortcut to appear in. This should be a list sepperated by &quot;;&quot; and be sure to end the list with this character as well</string>
     </property>
     <property name="text">
      <string>Games;Wine;</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QComboBox" name="prefixComboBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Executable:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Starts in:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="executablePushButton">
     <property name="text">
      <string>Click to change</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Categories:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QPushButton" name="cdPushButton">
     <property name="text">
      <string>Click to change</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="acceptPushButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="iconLineEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="browseIconToolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="nickLineEdit">
     <property name="placeholderText">
      <string>Nickname</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Prefix:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Icon:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="iconLabel">
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>312</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
